src/prescriptions/prescription-actions.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PrescriptionPrintAction from '../print-prescription/prescription-print-action.component';
+import styles from './prescriptions-actions.scss';
+import { Layer } from '@carbon/react';
+type PrescriptionsActionsFooterProps = {
+  encounterUuid: string;
+  patientUuid: string;
+};
+
+const PrescriptionsActionsFooter: React.FC<PrescriptionsActionsFooterProps> = ({ encounterUuid, patientUuid }) => {
+  return (
+    <Layer className={styles.actionsContainer}>
+      <div className={styles.actionsContainer}>
+        {/* Left buttons */}
+        <PrescriptionPrintAction encounterUuid={encounterUuid} patientUuid={patientUuid} />
+      </div>
+
+      <div className={styles.actionsContainer}>{/* Right buttons */}</div>
```

Comment:
Bit strange that the parent and child components all use the same CSS class. Maybe it should just be applied to one?

Reply:
Yeah, children should be actionCluster,i had already defined the class in scss but failed to use them by mistake.Fixed.Thanks